pages/web-lit-basics-two.jsx

```diff
@@ -0,0 +1,143 @@
+        {
+          title: "#allthestickerz",
+          image1x: "/img/pages/web-lit-basics-two/allthestickerz.png",
+          originalImgSrc: "https://d157rqmxrxj6ey.cloudfront.net/chadsansing/7048/stickerz-love.png ",
+          subtitle: "Understanding Sharing",
+          description: "Learners will create pixel art online/digital stickers, publish them for others, and use them to annotate and remix the Web, learning <strong>community participation, composing, open practices, remix, and sharing</strong>.",
+          link: "http://chadsansing.github.io/curriculum-testing/web-lit-basics-two/session04-all-the-stickerz.html"
+        },
+        {
+          title: "Who Am I?",
+          image1x: "/img/pages/web-lit-basics-two/who-am-i.png",
+          originalImgSrc: "http://chadsansing.github.io/curriculum-testing/web-lit-basics-two/question-mark.png",
+          subtitle: "Understanding Search",
+          description: "Learners will conduct a reverse image search to find information about a subject online and then revise a webpage with their own text and images, learning <strong>composing and search</strong>.",
+          link: "http://chadsansing.github.io/curriculum-testing/web-lit-basics-two/bridge03-who-am-i.html"
+        },
+        {
+          title: "Fair Use Free-for-All",
+          image1x: "/img/pages/web-lit-basics-two/fair-use-free-for-all.png",
+          originalImgSrc: "https://upload.wikimedia.org/wikipedia/commons/thumb/8/8b/Copyleft.svg/512px-Copyleft.svg.png",
+          subtitle: "Understanding Sharing",
+          description: "Learners will compete to identify examples and non-examples of fair use in peers' web remixes, learning <strong>credibility, search, and sharing</strong>.",
+          link: "http://chadsansing.github.io/curriculum-testing/web-lit-basics-two/session05-fair-use-free-for-all.html"
+        },
+        {
+          title: "The Planets",
+          image1x: "/img/pages/web-lit-basics-two/the-planets.jpg",
+          originalImgSrc: "https://upload.wikimedia.org/wikipedia/commons/thumb/d/da/NGC_3521-_the_Bubble_galaxy.jpg/320px-NGC_3521-_the_Bubble_galaxy.jpg",
+          subtitle: "Understanding Designing for Accessibility",
+          description: "Learners will improve the accessibility of a webpage by changing its color scheme, content, and embedded media, learning <strong>composing, designing for accessibility, and remixing</strong>.",
+          link: "http://chadsansing.github.io/curriculum-testing/web-lit-basics-two/session06-the-planets-and-accessibility.html"
+        }
+      ]
+    }
+  ],
+  render: function () {
+    var blogPostLink = "https://blog.webmaker.org/help-us-get-local-with-web-literacy";
+    return (
+      <div className="inner-container">
+        <CurriculumIntro/>
+        <WebLitBasics/>
+        {this.curriculum.map(function (section, key) {
+          return (
+            <ActivitySection title={section.title} key={key} activities={section.activities} />
```

Comment:
keys are for identifying items, so we can't use the array position. The title is a good unique identifier, though, so we can use `key={section.title}`